test/README.md

```diff
@@ -58,10 +58,19 @@ environment.
    See [Installing Helm](https://helm.sh/docs/intro/install/) for other options.
    Tested with version v3.11.
 
-1. Install the `virtctl` tool. See
+1. Install the `virtctl` tool.
+   Working Example:
+   ```
+   curl -L -O https://github.com/kubevirt/kubevirt/releases/download/v1.2.0/virtctl-v1.2.0-linux-amd64
+   ```
```

Comment:
The linter is not happy with this:
=====  mdl =====
test/README.md:63: MD031 Fenced code blocks should be surrounded by blank lines
test/README.md:65: MD031 Fenced code blocks should be surrounded by blank lines
test/README.md:70: MD031 Fenced code blocks should be surrounded by blank lines

Further documentation is available for these failures:
 - MD031: https://github.com/markdownlint/markdownlint/blob/main/docs/RULES.md#md031---fenced-code-blocks-should-be-surrounded-by-blank-lines

You need to add the missing blank lines around the code block.